app/api/helpers/payment.py

```diff
@@ -40,7 +40,14 @@ def get_credentials(event=None):
         """
         if not event:
             settings = get_settings()
-            if settings['stripe_secret_key'] and settings["stripe_publishable_key"] and settings[
+            if settings['app_environment'] == 'development' and settings['stripe_test_secret_key'] and \
+                settings['stripe_test_publishable_key'] and settings['stripe_test_secret_key'] != "" and \
+                    settings['stripe_test_publishable_key'] != "":
+                return {
+                    'SECRET_KEY': settings['stripe_test_secret_key'],
+                    'PUBLISHABLE_KEY': settings["stripe_test_publishable_key"]
+                }
+            elif settings['stripe_secret_key'] and settings["stripe_publishable_key"] and settings[
                 'stripe_secret_key'] != "" and \
                     settings["stripe_publishable_key"] != "":
```

Comment:
Redundant checks. Empty string is Falsy in python

Reply:
Sorry for the redundant check. I fixed it at another place where it was occurring